<?xml version="1.0"?>
<robot name="sensor_suite"> 
  <link name="os_sensor" />
  <link name="avia_frame" />
  <link name="mid360_frame" />
  <link name="cam_1_depth_optical_frame" />
  <link name="opti_markers" /> 

  <joint name="joint1" type="fixed">
    <parent link="avia_frame"/>
    <child link="os0_sensor"/>
    <origin rpy="0 0 0.7754" xyz="0.0 -0.35 0.1"/>
  </joint>
 
  <joint name="joint2" type="fixed">
    <parent link="avia_frame"/>
    <child link="os1_sensor"/>
    <origin rpy="0 0 -0.8" xyz="0.07 0.2 0.1"/> 
  </joint>
 
  <joint name="joint3" type="fixed">
    <parent link="avia_frame"/>
    <child link="velodyne"/>
    <origin rpy="0 0.06 -0.07" xyz="-0.2 -0.2 0.3"/>
  </joint>

  <joint name="joint4" type="fixed">
    <parent link="avia_frame"/>
    <child link="cam_1_depth_optical_frame"/>
    <origin rpy="0 0 0" xyz="0.0 0.1 0.1"/>
  </joint>
  
  <joint name="joint5" type="fixed">
    <parent link="avia_frame"/>
    <child link="horizon_frame"/>
    <origin rpy="0  0  -0.02" xyz="0 -0.16 0"/>
  </joint>

  <joint name="joint6" type="fixed">
    <parent link="avia_frame"/>
    <child link="opti_markers"/>
    <origin rpy="0 0.3 1.4324" xyz="-0.2 0.2 0.4"/>
  </joint>

 
</robot>  
